ocs_ci/deployment/disconnected.py

```
@@ -225,7 +264,14 @@ def prune_and_mirror_index_image(
 
 
 @retry((CommandFailed, NotFoundError), tries=3, delay=10, backoff=2)
-def mirror_index_image_via_oc_mirror(index_image, packages, idms=None):
+def mirror_index_image_via_oc_mirror(
+    index_image,
+    packages=None,
+    idms=None,
+    idms_name_prefix="odf",
+    configure_registries=False,
```

Ruff (0.15.13)

[warning] Boolean default positional argument in function definition

(FBT002)